permissions/Audio.permission

```diff
@@ -2,9 +2,13 @@
 
 # x-sailjail-translation-catalog = sailjail-permissions
 # x-sailjail-translation-key-description = permission-la-audio
-# x-sailjail-description = Play audio
+# x-sailjail-description = Play and record audio
 # x-sailjail-translation-key-long-description = permission-la-audio_description
-# x-sailjail-long-description = Play audio and show audio controls on lockscreen
+# x-sailjail-long-description = Play and record audio, and show audio controls on lockscreen
+
+### Do not prevent access to Pulseaudio socket etc. audio devices
+# This gives access also to microphone but for future compatibility use of Microphone is recommended as well.
```

Comment:
Perhaps this could more along the lines of

> While this allows access to microphone, you should require also Microphone permission to ensure access.

If you want, there could "subject to change" or that kind of expressions too.

Reply:
And of course if you want to explain it further to developer, you could say that pulse audio doesn't have means to control them separately.

Reply:
Added a lengthy description with a `FIXME` in 36f9ad3 as that separation of Alsa and Pulse permissions seemed to be a viable option in the future.